ci-operator/config/sclorg/ansible-tests/sclorg-ansible-tests-master__dotnet-ocp4.14-lp-interop.yaml

```diff
@@ -57,6 +57,35 @@ tests:
     test:
     - ref: dotnet-deploy-and-test
     workflow: firewatch-ipi-aws
+- as: dotnet-interop-ibmcloud
+  cron: 0 6 25 12 *
+  steps:
+    cluster_profile: ibmcloud-cspi-qe
+    env:
+      BASE_DOMAIN: zstream.interop.ccitredhat.com
+      DOTNET_VERSION: dotnet_60
```

Comment:
Can this be moved out of config to dockerfile or somewhere else

Reply:
We talking about the DOTNET_VERSION... Required env by the step.